pkg/plugins/scms/github/pullrequest.go

```diff
@@ -49,7 +49,7 @@ const PULLREQUESTBODY = `
 
 This pull request was automatically created using [Updatecli](https://www.updatecli.io).
 
-Please report any issues with this tool [here](https://github.com/updatecli/updatecli/issues/new)
+Please report any issues with this tool [here](https://github.com/updatecli/updatecli/issues/)
```

Comment:
```suggestion
Please report any issues with this tool [here](https://github.com/updatecli/updatecli/issues/new/choose)
```
Alternatively, link to the issue template chooser.
I think linking to issues list is better as users can see existing issues before opening a new one.